internal/report/report.go

```diff
@@ -239,7 +240,7 @@ func (rpt *Report) newGraph(nodes graph.NodeSet) *graph.Graph {
 	// Clean up file paths using heuristics.
 	prof := rpt.prof
 	for _, f := range prof.Function {
-		f.Filename = trimPath(f.Filename)
+		f.Filename = trimPath(f.Filename, o.TrimPath, "" /* only trim if the trim path is explicitly specified */)
```

Comment:
I think we should pass o.SourcePath here. 
Why don't we? Comment isn't very helpful.

Reply:
Done. My main motivation was to keep the behavior here as is since I was a little uncertain whether trimming the paths to display on the graph based on the search_path-based heuristics is a good idea or not. But passing o.SourcePath here makes the trimming here less special which is a good thing so done.